ideprobe/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:11
```

Comment:
Note that the plugin built by Pants is still via JDK 8, in case any discrepancy occurs in the future.